modules/button/react/README.md

```diff
@@ -760,6 +761,8 @@ Default: `undefined`
 
 ---
 
+# Toolbar Buttons
+
```

Comment:
```suggestion
```
Nit: I don't think this is necessary, especially when it's at the same header level as the components themselves